src/picongpu/include/particles/bremsstrahlung/ScaledSpectrum.hpp

```diff
@@ -0,0 +1,312 @@
+ */
+struct ScaledSpectrum
+{
+public:
+    typedef detail::LookupTableFunctor LookupTableFunctor;
+private:
+
+    typedef boost::shared_ptr<PMacc::container::DeviceBuffer<float_X, DIM2> > MyBuf;
+    MyBuf dBufScaledSpectrum;
+    MyBuf dBufStoppingPower;
+
+    /** differential cross section: cross section per unit energy
+     *
+     * This is the screened Bethe-Heitler cross section. See e.g.:
+     * Salvat, F., et al. "Monte Carlo simulation of bremsstrahlung emission by electrons."
+     * Radiation Physics and Chemistry 75.10 (2006): 1201-1219.
+     *
+     * @param Ekin kinetic electron energy
+     * @param kappa energy loss normalized to Ekin
+     */
+    float_64 dcs(const float_64 Ekin, const float_64 kappa) const
+    {
+        namespace pam = PMacc::algorithms::math;
+
+        const float_64 bohrRadius = 4.0 * M_PI * EPS0 * HBAR*HBAR / (ELECTRON_MASS * ELECTRON_CHARGE*ELECTRON_CHARGE);
+        const float_64 classicalElRadius = ELECTRON_CHARGE*ELECTRON_CHARGE / (4.0 * M_PI * EPS0 * ELECTRON_MASS * SPEED_OF_LIGHT*SPEED_OF_LIGHT);
+        const float_64 fineStructureConstant = ELECTRON_CHARGE*ELECTRON_CHARGE / (4.0 * M_PI * EPS0 * HBAR * SPEED_OF_LIGHT);
+
+        const float_64 c = SPEED_OF_LIGHT;
+        const float_64 c2 = c*c;
+        const float_64 m_e = ELECTRON_MASS;
+        const float_64 r_e = classicalElRadius;
+        const float_64 alpha = fineStructureConstant;
+
+        const float_64 W = kappa * Ekin;
+        const float_64 eps = W / (Ekin + m_e * c2);
+        const float_64 R = pam::pow(static_cast<float_64>(ion::TARGET_Z), -1.0/3.0) * bohrRadius;
+        const float_64 gamma = Ekin / (m_e * c2) + 1.0;
+        const float_64 b = R * m_e * c / HBAR / (2.0 * gamma) * eps / (1.0 - eps);
+
+        const float_64 phi_1 = 4.0 * pam::log(R * m_e * c / HBAR) + 2.0 - 2.0 * pam::log(1.0 + b*b) - 4.0 * b * pam::atan2(1.0 / b, 1.0);
+        const float_64 phi_2 = 4.0 * pam::log(R * m_e * c / HBAR) + 7.0/3.0 - 2.0 * pam::log(1.0 + b*b) - 6.0 * b * pam::atan2(1.0 / b, 1.0) \
+                             - b*b * (4.0 - 4.0 * b * pam::atan2(1.0 / b, 1.0) - 3.0 * pam::log(1.0 + 1.0 / (b*b)));
+
+        return r_e*r_e * alpha * ion::TARGET_Z*ion::TARGET_Z / W * (eps*eps * phi_1 + 4.0/3.0 * (1.0 - eps) * phi_2);
+    }
+
+    /** Scaled differential cross section
+     *
+     * = differential cross section times energy loss
+     *
+     * @param Ekin kinetic electron energy
+     * @param kappa energy loss normalized to Ekin
+     */
+    float_64 chi(const float_64 Ekin, const float_64 kappa) const
+    {
+        namespace pam = PMacc::algorithms::math;
+        /*
+        const float_64 c2 = SPEED_OF_LIGHT*SPEED_OF_LIGHT;
+        const float_64 m_e = ELECTRON_MASS;
+
+        const float_64 beta = pam::sqrt(Ekin * (Ekin + 2.0 * m_e * c2)) / (Ekin + m_e * c2);*/
+        const float_64 W = kappa * Ekin;
+
+        return /*beta*beta / (ion::TARGET_Z*ion::TARGET_Z) **/ W * this->dcs(Ekin, kappa);
```

Comment:
but than this is not chi anymore - please find a better/other name or go back to the original version